tests/integration/sender_integration_test.go

```diff
@@ -97,19 +111,27 @@ func setupQuestDB0(ctx context.Context, auth ilpAuthType, setupProxy bool) (*que
 	env := map[string]string{
 		"QDB_CAIRO_MAX_UNCOMMITTED_ROWS":        "1",
 		"QDB_LINE_TCP_MAINTENANCE_JOB_INTERVAL": "100",
-		"QDB_PG_ENABLED":                        "false",
+		"QDB_PG_ENABLED":                        "true",
```

Comment:
nit: do we use PGWire in tests?

Reply:
We don't directly, but the container healthchecks fail when `QDB_PG_ENABLED=false`:
```
2024-03-05T19:52:40.664738Z I i.q.c.h.p.StaticContentProcessor [96] incoming [url=/]
2024-03-05T19:52:41.664923Z I http-server scheduling disconnect [fd=96, reason=12]
2024-03-05T19:52:41.664952Z I http-server disconnected [ip=10.89.3.36, fd=96, src=queue]
```

Reply:
Hmm, that's unexpected. Could you check with a standalone Docker container? Maybe we have a bug on the server side.